lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -1502,18 +1502,7 @@ protected function readEditionLayers(QgisProject $xml)
             return;
         }
 
-        $editionLayers = $this->cfg->getEditionLayers();
-
-        // Check ability to load spatialite extension
-        // And remove ONLY spatialite layers if no extension found
-        $spatialiteExt = '';
-        if (class_exists('SQLite3')) {
-            $spatialiteExt = $this->getSpatialiteExtension();
```

Comment:
We should probably get rid of this method too (and search for "spatialite" or "sqlite" in the code too)

Reply:
sqlite can still be used for database backend for Lizmap users and groups.

* https://github.com/3liz/lizmap-web-client/search?q=spatialite
* https://github.com/3liz/lizmap-web-client/search?q=sqlite

If you want further cleaning, I guess a note should be added in separate PR, related to https://github.com/3liz/lizmap-web-client/issues/2058 ?